pyglmnet/pyglmnet.py

```diff
@@ -541,7 +543,8 @@ def __init__(self, distr='poisson', alpha=0.5,
                  learning_rate=2e-1, max_iter=1000,
                  tol=1e-6, eta=2.0, score_metric='deviance',
                  fit_intercept=True,
-                 random_state=0, callback=None, verbose=False):
+                 random_state=0, callback=None, verbose=False,
+                 show_progress=True):
```

Comment:
what's the difference between `verbose` and `show_progress`? Can we have `verbose` levels like in joblibs or scikit-learn?

Reply:
personally, I see `verbose` more like a utility to print more information about the status of the system (e.g., from simple status messages to debug messages). `show_progress` refers more specifically to give a better user experience by formatting in a nice way the training progress information.

No hard feelings on this though.